### ISSUE TITLE
Redirection Error - JSON Parse error: Unrecognized token '<'

### ISSUE THREAD

**Issue by thijsdevries** [johngodley/redirection#660]
```
Plugin: 3.0.1
WordPress: 4.8.5
PHP: 5.5.38
Browser: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_2) AppleWebKit/604.4.7 (KHTML, like Gecko) Version/11.0.2 Safari/604.4.7

Error: JSON Parse error: Unrecognized token '<'
Action: redirection/v1/redirect?orderby=id GET
Code: 404 Not Found
Raw: <!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html><head>
<title>404 Not Found</title>
</head><body>
<h1>Not Found</h1>
<p>The requested URL /wp-json/redirection/v1/redirect was not found on this server.</p>
</body></html>


Error: JSON Parse error: Unrecognized token '<'
Action: redirection/v1/group GET
Code: 404 Not Found
Raw: <!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html><head>
<title>404 Not Found</title>
</head><body>
<h1>Not Found</h1>
<p>The requested URL /wp-json/redirection/v1/group was not found on this server.</p>
</body></html>

```

**Comment by thijsdevries** [comment]
Anybody know how I could fix this?

**Comment by johngodley** [comment]
Can you try the 3.1 beta?

https://github.com/johngodley/redirection/releases/tag/3.1

**Comment by johngodley** [comment]
I've now released version 3.1. I will close this ticket for now, but please reopen if the problem persists
